spec/models/item_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+describe Item do
+  describe ".for_category" do
+    it "returns items for_category" do
+      category = categories(:flip_flops)
+      expect(Item.unscoped.where(category_id: category.id).count).to eq(4)
```

Comment:
This and the next example should be using `aggregate_failures`, I think

Reply:
not a pattern we currenly use, could do this later